tests/unittests/interface_types.cpp

```diff
@@ -61,6 +61,29 @@ TEST_CASE("InterfaceTypes static checks", "[interface-types][static-checks]") {
   STATIC_REQUIRE(TypeWithEnergy::typeName == "TypeWithEnergy"sv);
 }
 
+TEST_CASE("InterfaceTypes hash", "[interface-types][hash]") {
+  auto hit = ExampleHit();
+  auto wrapper1 = TypeWithEnergy(hit);
+  auto hash1 = std::hash<TypeWithEnergy>{}(wrapper1);
+
```

Comment:
```suggestion
  REQUIRE(hash1 == std::hash<ExampleHit>{}(hit));
```
Just to be very clear and documenting about this. (I am assuming that the hashes are equal here, but it doesn't actually matter, we should just put this here explicitly).

Reply:
Good idea!